checkov/terraform/parser.py

```diff
@@ -642,6 +711,36 @@ def get_next_vertices(evaluated_files: list[str], unevaluated_files: list[str])
             unevaluated.append(k)
         return next_level, unevaluated
 
+    @staticmethod
+    def get_nested_modules_data_as_list(file_path):
+        path = get_abs_path(file_path)
+        modules_list = []
+
+        while is_nested(file_path):
+            module, index = get_module_from_full_path(file_path)
+            modules_list.append((module, index))
+            file_path = module
+        return list(reversed(modules_list)), path
```

Comment:
```suggestion
            file_path = module
        modules_list.reverse()
        return modules_list, path
```
no need to create a new list